Misc/NEWS.d/next/Library/2018-10-29-04-14-58.bpo-35101.xImh8d.rst

```diff
@@ -0,0 +1,2 @@
+Fix inspect.findsource incorrectly returning the line number of the first
```

Comment:
That might be a bit unfair since more functions than just `getsource` use it. `getcomments` and `getsourcelines` use it directly while `getsource` uses `getsourcelines`, using it indirectly. Would it be OK to mention all of them?

Reply:
The last two commits ef1e7502256 and e4e811d65b5 used `getsource` in the NEWS though `findsource` was fixed since I think `findsource` is not documented. I think we can wait for the reviewer on this 👍